### ISSUE TITLE
add to flathub

### ISSUE THREAD

**Issue by RustoMCSpit** [stuffbydavid/Mine-imator#29]
flathub.org is the largest repo for linux apps

you have to PR it here  https://github.com/flathub/flathub/pulls

**Comment by supermarioscratch** [comment]
Yeah. i NEED a flathub release because the linux version are not working. i tested it on 3 distros.
